sopel/modules/admin.py

```diff
@@ -185,9 +185,9 @@ def quit(bot, trigger):
 @plugin.priority('low')
 @plugin.example('.say #YourPants Does anyone else smell neurotoxin?')
 def say(bot, trigger):
-    """
-    Send a message to a given channel or nick. Can only be done in privmsg by
-    an admin.
+    """Send a message to a given channel or nick.
+
+    Can only be done in privmsg by an admin.
```

Comment:
While we are at it, let's remove the possible confusion between `PRIVMSG` and "Private message":

```suggestion
    Can only be done in a private message by an admin.
```